commercetools/resource_state_test.go

```diff
@@ -43,6 +44,33 @@ func TestAccState_createAndUpdateWithID(t *testing.T) {
 					),
 				),
 			},
+			{
+				Config: testAccTransitionConfig(t, transition),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr(
+						"commercetools_state.acctest-t1", "transitions.#", "1",
+					),
+					resource.TestCheckResourceAttr(
+						"commercetools_state.acctest-t1", "transitions.0", transition,
```

Comment:
I've been trying to fix the testing framework (which requires updating the Python sdk). I cannot get this check to work. I tried this against an actual Commercetools environment as well, which doesn't work either. Can you remove this check?